app/templates/403.html

```diff
@@ -3,4 +3,6 @@
 {% block content %}
     <h1>Sorry, you don't have access to this page.</h1>
     <p>For security reasons, you must be verifiably listed in the Yale directory as an undergraduate in order to access Yalies, CourseTable, and other YCS projects. If you are in the directory but are still seeing this message, please email <a href="mailto:yalecomputersociety@gmail.com">yalecomputersociety@gmail.com</a> for help.</p>
+    <h2>Are you an incoming Yale student, who has not yet started their first term of classes?</h2>
+    <p>Access to Yalies.io and other Y/CS products are granted after matriculation, usually in August before classes start. Please see our <a href="https://yalecomputersociety.org/newstudents">matriculation access policy</a> for more information.</p>
```

Comment:
Super super minor but I tend to prefer that we just refer to it as "Yalies" rather than "Yalies.io" when we're writing it in text with capitalization and such and it's not actually a link.

Reply:
@ErikBoesen gotcha, I’m happy to push an edit to keep things consistent!